app/client/components/contactus/contactus.tsx

```diff
@@ -0,0 +1,388 @@
+import { css } from "@emotion/core";
+import { Button } from "@guardian/src-button";
+import { palette, space } from "@guardian/src-foundations";
+import { headline, textSans } from "@guardian/src-foundations/typography";
+import { RouteComponentProps } from "@reach/router";
+import React, { useEffect, useState } from "react";
+import { minWidth } from "../../styles/breakpoints";
+import { trackEvent } from "../analytics";
+import { contactUsConfig } from "./contactUsConfig";
+import { ContactUsForm } from "./contactUsForm";
+import { ContactUsHeader } from "./contactUsHeader";
+import { ContactUsPageContainer } from "./contactUsPageContainer";
+import { SelfServicePrompt } from "./selfServicePrompt";
+import { SubTopicForm } from "./subTopicForm";
+import { TopicButton } from "./topicButton";
+
+interface ContactUsFormStateSnapshot {
+  selectedTopic: string | undefined;
+  selectedSubTopic: string | undefined;
+  selectedSubSubTopic: string | undefined;
+}
+
+interface ContactUsProps extends RouteComponentProps {
+  topicId?: string;
+  subTopicId?: string;
+  subSubTopicId?: string;
```

Comment:
Seems strange to have everything optional.

Reply:
Hey Mario, we've renamed these props to better reflect what they do. They are only used if the user is deep linking into the form via the url and as such all of them are optional as you can use the form without deep linking.